packages/@okta/vuepress-site/docs/reference/api/schemas/index.md

```diff
@@ -2462,3 +2464,351 @@ Specific property types support a subset of [JSON Schema validations](https://to
+    }
+  }
+}' "https://${yourOktaDomain}/api/v1/meta/schemas/user/default"
+```
+
+##### Response example
+
+The following response is only a subset of properties for brevity.
+
+```json
+{
+    "id": "https://${yourOktaDomain}/meta/schemas/user/default",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "name": "user",
+    "title": "Default Okta User",
+    "lastUpdated": "2015-09-05T10:40:45.000Z",
+    "created": "2015-02-02T10:27:36.000Z",
+    "definitions": {
+        "base": {
+            "id": "#base",
+            "type": "object",
+            "properties": {
+                "login": {
+                    "title": "Username",
+                    "type": "string",
+                    "required": true,
+                    "minLength": 5,
+                    "maxLength": 100,
+                    "permissions": [
+                        {
+                            "principal": "SELF",
+                            "action": "READ_WRITE"
+                        }
+                    ]
+                },
+                "firstName": {
+                    "title": "First name",
+                    "type": "string",
+                    "required": true,
+                    "minLength": 1,
+                    "maxLength": 50,
+                    "permissions": [
+                        {
+                            "principal": "SELF",
+                            "action": "READ_WRITE"
+                        }
+                    ]
+                },
+                "lastName": {
+                    "title": "Last name",
+                    "type": "string",
+                    "required": true,
+                    "minLength": 1,
+                    "maxLength": 50,
+                    "permissions": [
+                        {
+                            "principal": "SELF",
+                            "action": "READ_WRITE"
+                        }
+                    ]
+                },
+                "email": {
+                    "title": "Primary email",
+                    "type": "string",
+                    "required": true,
+                    "format": "email",
+                    "permissions": [
+                        {
+                            "principal": "SELF",
+                            "action": "READ_WRITE"
+                        }
+                    ]
+                }
+            },
+            "required": [
+                "login",
+                "firstName",
+                "lastName",
+                "email"
+            ]
+        },
+        "custom": {
+            "id": "#custom",
+            "type": "object",
+            "properties": {
+              "twitterUserName": {
+                  "title": "Twitter username",
+                  "description": "User's username for twitter.com",
+                  "type": "string",
+                  "required": false,
+                  "unique": "UNIQUE_VALIDATED",
```

Comment:
Do we explain anywhere how requesting `"unique": true` results in this?

Reply:
Added explanation at L2486.